sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -113,14 +113,26 @@ statement
         LIKE source=tableIdentifier locationSpec?                      #createTableLike
     | ANALYZE TABLE tableIdentifier partitionSpec? COMPUTE STATISTICS
         (identifier | FOR COLUMNS identifierSeq | FOR ALL COLUMNS)?    #analyze
-    | ALTER TABLE tableIdentifier
-        ADD COLUMNS '(' columns=colTypeList ')'                        #addTableColumns
+    | ALTER TABLE multipartIdentifier
+        ADD (COLUMN | COLUMNS) columns=newColumnList colPosition?      #addTableColumns
```

Comment:
`colPosition` should be related to each column when you adding multiple new columns, right?

Reply:
Below is the example you can follow

```
    | ALTER TABLE tableIdentifier
        ADD COLUMNS '(' columns=qualifiedColTypeWithPositionList ')'   #addTableColumns

qualifiedColTypeWithPositionList
    : qualifiedColTypeWithPosition (',' qualifiedColTypeWithPosition)*
    ;

qualifiedColTypeWithPosition
    : (path=qualifiedName '.')? colType colPosition?
    ;

colType  <-- reuse the existing one
    : identifier dataType (COMMENT STRING)?
```

Reply:
@gatorsmile, I'll remove `colPosition` from `ADD`. It is not supported and was not previously parsed so it doesn't need to be included.

Also, `colType` can't be used because we need to use qualified names.

Reply:
I've updated this. I didn't use the original `colType` because it can't use the existing code to handle a qualified name.